lib/plugins/tablist.js

```diff
@@ -1,7 +1,7 @@
 module.exports = inject
 
 const escapeValueNewlines = str => {
-  return str.replace(/": *"((?:\\"|[^"])+)/, (_, match) => '":"' + match.replace('\n', '\\n'))
+  return str.replace(/(": *"(?:\\"|[^"])+")/g, (_, match) => match.replace('\n', '\\n'))
```

Comment:
Why ? What's the purpose ? Please explain in PR description

Reply:
I'm matching whole replaced string so I don't have to add `": "` later. Also I'm replacing every matched string which fixes #873